app/views/applications/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<h3>Application Currently <%=@application.status%></h3>
+<p><%=@application.name%></p>
+<p><%=@application.address%></p>
+<p><%=@application.description%></p>
+
+<p>
+    <% @application.pets.each do |pet| %>
+        <%= link_to "#{pet.name}", "/pets/#{pet.id}" %>
+    <% end %>
+</p>
+
+<section id="search">
+    <% if @application.status == "In Progress" %>
+        <%= form_with method: :get, local: true do |f| %>
+        <%= f.label "Add a Pet to this Application" %></br>
+        <%= f.text_field :search_app %>
+        <%= f.submit "Submit" %>
+        <% end %>
+    <% end %></br>
+</section>
+
+<% if @pets.present? %>
+    <% @pets.each do |pet| %>
+        <%= pet.name %> 
+        <%= button_to "Adopt this Pet", "/applications/#{@application.id}", method: :get, params: { pet_id: pet.id } %>
```

Comment:
when we click this button, should it be a GET request? Keep in mind we are *creating* a joins record between a pet and an application in this case, so we could probably use a POST here instead. Similar to the previous comment around paths for joins records, a more RESTful route here could be: "/applications/:app_id/pets" to indicate you are joining these 2 resources